cmd/mirror-main.go

```diff
@@ -541,13 +536,12 @@ func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch
 		isWatch:        isWatch,
 		excludeOptions: excludeOptions,
 
-		status:         status,
-		scanBar:        func(s string) {},
-		statusCh:       make(chan URLs),
-		queueCh:        make(chan func() URLs),
-		wgStatus:       new(sync.WaitGroup),
-		watcherRunning: true,
-		watcher:        NewWatcher(UTCNow()),
+		status:   status,
+		scanBar:  func(s string) {},
+		statusCh: make(chan URLs),
+		queueCh:  make(chan func() URLs),
+		wgStatus: new(sync.WaitGroup),
```

Comment:
is `watcherRunning` member needed elsewhere or can it be removed from `mirrorJob`?

Reply:
It can be removed not needed.